v2.0/demo-json-support.md

```diff
@@ -0,0 +1,186 @@
+---
+title: JSON Support
+summary: Use a local cluster to explore how CockroachDB can store and query unstructured JSONB data.
+toc: false
+---
+
+<span class="version-tag">New in v2.0:</span> CockroachDB now supports [`JSONB`](jsonb.html) columns.
+
+This page walks you through a simple demonstration of how CockroachDB can store and query unstructured `JSONB` data from a third-party API, as well as how an [inverted index](inverted-indexes.html) can optimize your queries.
+
+<div id="toc"></div>
+
+## Step 1. Install prerequisites
+
+- Install the latest version of [CockroachDB](install-cockroachdb.html).
+- Install the latest version of [Go](https://golang.org/dl/): `brew install go`
+
+## Step 2. Start a single-node cluster
+
+For the purpose of this tutorial, you need only one CockroachDB node running in insecure mode:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach start \
+--insecure \
+--store=hello-1 \
+--host=localhost
+~~~
+
+## Step 3. Create a user
+
+In a new terminal, as the `root` user, use the [`cockroach user`](create-and-manage-users.html) command to create a new user, `maxroach`.
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach user set maxroach --insecure
+~~~
+
+## Step 4. Create a database and grant privileges
+
+As the `root` user, open the [built-in SQL client](use-the-built-in-sql-client.html):
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure
+~~~
+
+Next, create a database called `jsonb_test`:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE DATABASE jsonb_test;
+~~~
+
+Set the database as the default:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET DATABASE = jsonb_test;
+~~~
+
+Then [grant privileges](grant.html) to the `maxroach` user:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> GRANT ALL ON DATABASE jsonb_test TO maxroach
+~~~
+
+## Step 5. Create a table
+
+In the [built-in SQL client](use-the-built-in-sql-client.html), create a table called `programming`:
```

Comment:
nit: Intro to this sentence feels redundant. Maybe something like:

```
Still in the SQL shell, create a table called `programming`:
```

Reply:
done